classes/submission/Collector.inc.php

```diff
@@ -367,7 +367,7 @@ public function getQueryBuilder(): Builder
 
         // Add app-specific query statements
         HookRegistry::call('Submission::Collector', [&$q, $this]);
-
+error_log($q->toSql());
```

Comment:
@asmecher Just spotted this debug while rebasing :)

Reply:
Reverted in https://github.com/pkp/pkp-lib/commit/db818df0063b78adf14b75370b7682a3cf68f394.